appveyor.yml

```diff
@@ -25,7 +25,7 @@ before_build:
   - cd c:\projects\uncrustify
   - mkdir build
   - cd build
-  - cmake -DCMAKE_BUILD_TYPE=%CONFIGURATION% -DUNCRUSTIFY_SEPARATE_TESTS=ON -G "%GENERATOR%" ..
+  - cmake -DUNCRUSTIFY_SEPARATE_TESTS=ON -G "%GENERATOR%" ..
```

Comment:
sorry not to have replied in due time but why have you removed setting the built type? This will result making only Debug or Release (whichever is the default).

Reply:
did you actually wanted to do this https://github.com/uncrustify/uncrustify/pull/2990 instead?